common/autoware_ad_api_msgs/planning/msg/SteeringFactor.msg

```diff
@@ -0,0 +1,29 @@
+# constants for common use
+uint16 UNKNOWN = 0
+
+# constants for type
+uint16 INTERSECTION = 1
+uint16 LANE_CHANGE = 2
+uint16 AVOIDANCE_PATH_CHANGE = 3
+uint16 AVOIDANCE_PATH_RETURN = 4
+uint16 STATION = 5
+uint16 PULL_OUT = 6
+uint16 PULL_OVER = 7
+uint16 EMERGENCY_OPERATION = 8
+
+# constants for direction
+uint16 LEFT = 1
+uint16 RIGHT = 2
```

Comment:
In the case of pull_out, the ego drives backward and then departing. Is it correct to use the same right direction when driving backward as when departing?
Or it's better to prepare `STRAIGHT`?

![image](https://user-images.githubusercontent.com/39142679/189830125-1e474407-ece8-4462-ad6d-0156097755bf.png)

Reply:
I add STRAIGHT in this commit [da2db2d](https://github.com/autowarefoundation/autoware.universe/pull/1706/commits/da2db2d0f9050bd975ad170a0e7f98747b17fb97)
@kosuke55 May I know how can I reproduce this scenario?

Reply:
thaks!

https://user-images.githubusercontent.com/39142679/189852815-c592cf11-a70a-4933-a4b5-b88ff6586a5f.mp4

Please put dummy object in front of the ego.

> May I know how can I reproduce this scenario?

Reply:
Thanks! It does change to straight with label 3 
![Screenshot from 2022-09-13 17-47-56](https://user-images.githubusercontent.com/41041286/189856484-c5da8e22-b74c-45cc-b4f3-e007bc91bb69.png)

Reply:
hmm, In my case the label is 3

before starting backward driving:  label3
![image](https://user-images.githubusercontent.com/39142679/189860354-8d5809bd-8627-486c-b666-78593af37247.png)

during backward driving:  label3
![image](https://user-images.githubusercontent.com/39142679/189860441-6f775728-77d0-4c63-8e80-af08dfba41e6.png)

after finishing backward driving: label2
![image](https://user-images.githubusercontent.com/39142679/189860524-abfd46f3-3b76-4a0e-8a78-8b7778e8faaf.png)

Reply:
Sorry, I misunderstood. It's the same. Looks good!

Reply:
@kosuke55 
Yes. straight is labeled as 3.
So until any turn signal comes out, it will output straight.